lib/src/base/paged_layout_builder.dart

```diff
@@ -172,7 +172,8 @@ class _PagedLayoutBuilderState<PageKeyType, ItemType>
       covariant PagedLayoutBuilder<PageKeyType, ItemType> oldWidget) {
     super.didUpdateWidget(oldWidget);
     if (oldWidget.state != widget.state) {
-      if (_state.status == PagingStatus.loadingFirstPage) {
+      if (_state.status == PagingStatus.loadingFirstPage
```

Comment:
I printed oldWidget.state and widget.state 

```console
I/flutter (15329): PagingStateBase<int, String>(pages: null, keys: null, error: null, hasNextPage: true, isLoading: false)
I/flutter (15329): PagingStateBase<int, String>(pages: null, keys: null, error: null, hasNextPage: true, isLoading: true)
```

As you can see,  _fetchNextPage() will call again _fetchNextPage() when isLoading is true.